test/karma.conf.js

```diff
@@ -1,71 +1,28 @@
-module.exports = function(config) {
-  var customLaunchers = {
-    chrome_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'chrome',
-      platform: 'Windows 8.1',
-      version: '34'
-    },
-
-    firefox_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'firefox',
-      platform: 'Linux',
-      version: '29'
-    },
-
-    safari_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'safari',
-      platform: 'OS X 10.8'
-    },
-
-    ipad_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'ipad',
-      platform:'OS X 10.9',
-      version: '7.1'
-    },
-
-    android_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'android',
-      platform:'Linux'
-    },
-
-    ie_sl: {
-      singleRun: true,
-      base: 'SauceLabs',
-      browserName: 'internet explorer',
-      platform: 'Windows 8.1',
-      version: '11'
-    }
-  };
+var customLaunchers;
 
+module.exports = function(config) {
   config.set({
     basePath: '',
 
-    frameworks: ['qunit'],
-
+    frameworks: ['browserify', 'qunit'],
     autoWatch: false,
-
     singleRun: true,
 
     files: [
-      // include and execute the standalone test bundle first
-      '../build/temp/tests.js',
-
-      // then include video.js through globals to run the API tests
-      '../build/temp/video-js.min.css',
-      '../build/temp/video.min.js',
-      'api/api.js'
+      '../build/temp/video-js.css',
+      '../test/globals-shim.js',
+      '../test/unit/**/*.js'
     ],
 
+    preprocessors: {
+      '../test/**/*.js': [ 'browserify' ]
```

Comment:
is this thing actually working well? We've had trouble with it previously. Also, it makes it annoying to work with the standalone qunit file. At the very least, we'd want a separate watchify task to build out the test file that the standalone html page wants.